### ISSUE TITLE
tests in a child theme

### ISSUE THREAD

**Issue by aschweigert** [INN/Largo-Sample-Child-Theme#4]
would this even work? do we know?

**Comment by wjhdev** [comment]
Yeah. Using @rnagle's test scaffolding in deploy tools, running tests in a theme or child theme should be the same as running tests under Largo.

**Comment by rnagle** [comment]
Added a bunch of details [here](https://github.com/INN/Largo-Sample-Child-Theme/blob/master/tests/readme.md).

I used the instructions I wrote to make sure the tests included in the sample theme would run and pass.

The copy could probably use an edit.
